src/components/media-types/html/index.js

```diff
@@ -133,7 +133,7 @@ export const HTMLComponent = (props) => {
           <div className={styles.button}>
             <Button>
               <VisuallyHidden>View</VisuallyHidden>
-              {/*               <svg
```

Comment:
@melMass do you know why this was commented out?

Reply:
I'm going to answer on discord 😅